apps/api/src/billing/services/stripe/stripe.service.ts

```diff
@@ -144,6 +146,11 @@ export class StripeService extends Stripe {
     metadata?: Record<string, string>;
   }): Promise<{ success: boolean; paymentIntentId?: string }> {
     const discounts = await this.getCustomerDiscounts(params.customer);
+
+    if (!discounts.length && params.amount < MINIMUM_PAYMENT_AMOUNT) {
+      throw new Error(`Minimum payment amount is $${MINIMUM_PAYMENT_AMOUNT} (before any discounts)`);
```

Comment:
I'm not sure I follow, why not putting a limit of 20 in the zod schema instead of this check?

Reply:
I was planning on removing this logic of discounts in the payment intent entirely in another pr. I'll move it to zod and remove this check for now..